app/graphql/types/doi_item.rb

```diff
@@ -909,7 +917,7 @@ def citeproc_hsh
       "volume" => object.container.to_h["volume"],
       "issue" => object.container.to_h["issue"],
       "page" => page,
-      "publisher" => object.publisher,
+      "publisher" => object.publisher["name"] || object.publisher,
```

Comment:
The accessor will always return a hash so object.publisher will always be a hash.  However, it can return nil if publisher has never been set, which is possible - in draft dois.  Findable or registered dois should always have at least a name.

@codycooperross should take a look at this since he knows more about graphql.

Reply:
Actually, I realize that there is a case where publisher can be nil, so better check for that.

Reply:
Ultimately we implemented Ashwini's suggestion.